QR Extractor.py

```diff
@@ -0,0 +1,20 @@
+from qreader import QReader
+from cv2 import QRCodeDetector, imread
+from pyzbar.pyzbar import decode
+
+# Initialize the three tested readers (QRReader, OpenCV and pyzbar)
+qreader_reader, cv2_reader, pyzbar_reader = QReader(), QRCodeDetector(), decode
+
```

Comment:
If you want it to be a real extractor script, create a QRExtractor class with an extract method.
Its init should get a file path.
use argparse module to get the input for the user.